File: AngularSon-master/src/app/features/models/loanTransaction.ts
export interface LoanTransaction {
    id: string;
    memberId: string;
    bookId: string;
    bookName:string;
    returnStatus: ReturnStatus;
    returnTime: Date;
    
}
export enum ReturnStatus {
    NotReturned = 1,
    Returned = 2,
    Borrowed = 3
}

File: C-_Son-master/src/deneme2/Domain/Entities/LoanTransaction.cs
using Domain.Enums;
using NArchitecture.Core.Persistence.Repositories;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Domain.Entities;
public class LoanTransaction:Entity<Guid>
{
    public Guid MemberId { get; set; }
    public Guid BookId { get; set; }
    public ReturnStatus ReturnStatus { get; set; }
    public DateTime ReturnTime { get; set; }

    public virtual Member? Member { get; set; }
    public virtual Book? Book { get; set; }

}
